storm-server/src/test/java/org/apache/storm/scheduler/resource/strategies/scheduling/TestLargeCluster.java

```diff
@@ -70,6 +70,8 @@
 
     public static final String TEST_CLUSTER_NAME = "largeCluster01";
     public static final String TEST_RESOURCE_PATH = "clusterconf/" + TEST_CLUSTER_NAME;
+    public static final String COMPRESSED_SERIALIZED_TOPOLOGY_FILENAME_ENDING = "code.ser";
```

Comment:
There is a variable with the same name in TestTopologyAnonymizerUtils.java but with different meanings. What's the difference here?

Reply:
Anonymizer can be stricter with naming convention.  Files in the resources need just the bare minimum to be distinguished from each other. But more importantly, they are independent of each other.